.github/workflows/release.yml

```diff
@@ -21,14 +22,26 @@ jobs:
         name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: "1.21"
+          go-version: "1.16"
```

Comment:
Were there issues with the go version here being set to 1.21 and the version in go.mod being different (it's 1.16 there)?